corehq/apps/locations/tests/test_location_queries.py

```diff
@@ -171,3 +155,47 @@ def test_filter_path_by_user_input(self):
             .accessible_to_user(self.domain, self.web_user)
         )
         self.assertItemsEqual([], no_locs)
+
+
+class TestFilterPath(LocationHierarchyTestCase):
+    location_type_names = ['state', 'county', 'city']
+    location_structure = [
+        ('Massachusetts', [
+            ('Middlesex', [
+                ('Cambridge', []),
+                ('Somerville', []),
+                ('Evil Somerville', []),
+            ]),
+            ('Suffolk', [
+                ('Boston', []),
+                ('Brookline', []),
+            ])
+        ]),
+        ('Copycat State', [
```

Comment:
[Pack the Union](https://harvardlawreview.org/2020/01/pack-the-union-a-proposal-to-admit-new-states-for-the-purpose-of-amending-the-constitution-to-ensure-equal-representation/) :us: :eagle: :statue_of_liberty: